app/src/ui/create-tag/create-tag-dialog.tsx

```diff
@@ -100,6 +84,39 @@ export class CreateTag extends React.Component<
     )
   }
 
+  private renderPreviousTags() {
+    if (!enablePreviousTagSuggestions()) {
+      return null
+    }
+
+    const { localTags } = this.props
+    const { previousTags, tagName } = this.state
+
+    if (previousTags === null || localTags === null || localTags.size === 0) {
+      return null
+    }
+
+    const title = __DARWIN__ ? 'Previous Tags' : 'Previous tags'
+    const lastThreeTags = previousTags.slice(-3)
+
+    return (
+      <>
+        <p>{title}</p>
+        {lastThreeTags.length === 0 ? (
+          <>
```

Comment:
Small nit - I don't believe the <> </> are needed because the paragraph is self-encapsulating.